Description: Relax test dependencies
 Update criterion to the version currently in Debian and drop wasm-related deps
 as we don't target this arch.
Author: Arnaud Ferraris <aferraris@debian.org>
Forwarded: not-needed
Last-Update: 2023-11-21
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: a/Cargo.toml
===================================================================
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -64,7 +64,7 @@
 version = "0.3"
 
 [dev-dependencies.criterion]
-version = "0.4"
+version = "0.5"
 
 [dev-dependencies.quickcheck]
 version = "1.0.0"
@@ -80,11 +80,5 @@
     "serde_derive",
 ]
 
-[target."cfg(target_arch = \"wasm32\")".dependencies.js-sys]
-version = "0.3.56"
-
-[target."cfg(target_arch = \"wasm32\")".dev-dependencies.wasm-bindgen-test]
-version = "0.3"
-
 [badges.maintenance]
 status = "passively-maintained"
